.github/workflows/check.yml

```
@@ -1,9 +1,8 @@
 name: "Test"
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)